content/en/continuous_integration/setup_tests/javascript.md

```diff
@@ -250,7 +256,7 @@ If you are running tests in non-supported CI providers or with no `.git` folder,
 [Mocha >=9.0.0][8] uses an ESM-first approach to load test files. That means that if ES modules are used (for example, by defining test files with the `.mjs` extension), _the instrumentation is limited_. Tests are detected, but there isn't visibility into your test. For more information about ES modules, see the [NodeJS documentation][9].
 
 ### Browser tests
-The JavaScript tracer does not support browsers, so browser tests with `mocha` or `jest` do not provide visibility on the browser. For tests with `cypress`, `dd-trace` provides visibility on the node process running the test, but not on the browser. 
+The JavaScript tracer does not support browsers, so browser tests with `mocha` or `jest` do not provide visibility on the browser. For tests with `cypress`, `dd-trace` provides visibility on the node process running the test, but not on the browser.
```

Comment:
```suggestion
As the JavaScript tracer cannot instrument browsers, you cannot use the `mocha` or `jest` integration 
to instrument browser tests. For tests with `cypress`, `dd-trace` provides visibility on the node process running the test, but not in the browser.
```

Reply:
this one is a bit difficult. I'm not convinced by either option. I'll try to come up with something better